p2p/pex/pex_reactor.go

```diff
@@ -100,6 +100,11 @@ func (r *PEXReactor) minReceiveRequestInterval() time.Duration {
 	return r.ensurePeersPeriod / 3
 }
 
+// isMaxDialPeriodTarget checks if the addr exponential backoff target for or not
+func (r *PEXReactor) isMaxDialPeriodTarget(addr *p2p.NetAddress) bool {
```

Comment:
this function mixes 2 distinct responsibilities. better to move it to p2p package or get rid of it competely

Reply:
That looks better. I'll do that. Thanks